index.html

```diff
@@ -37,12 +44,18 @@
     </nav>
 
     <header>
+
         <div class="jumbotron">
+
             <div class="container">
+
                 <h1>Pokaż nam co masz w lodówce, a
```

Comment:
tutaj zamiast dodawać niepotrzebne puste linie bardziej chodziło o to aby w jednej lini otworzyć h1, w drugiej trzymać treść, w trzeciej zamknąć h1

Reply:
czyli nie ma problemu (a nawet wskazane), żeby w linii był tylko '>'

Reply:
?

Reply:
<h1>
  Pokaż nam co masz w lodówce
</h1>